crates/lsp/src/language.rs

```diff
@@ -1,13 +1,37 @@
 use marzano_language::target_language::{PatternLanguage, TargetLanguage};
 
 pub fn language_id_to_pattern_language(language_id: &str) -> Option<PatternLanguage> {
```

Comment:
This is used specifically for vs code. It should not be used for CLI stuff... I am 90% sure your change didn't fix the issue / wasn't tested.